src/super_gradients/training/models/detection_models/pp_yolo_e/pp_yolo_e.py

```diff
@@ -81,6 +81,20 @@ def forward(self, inputs: Tuple[Tuple[Tensor, Tensor], Tuple[Tensor, ...]]) -> T
 
         return output_pred_bboxes, output_pred_scores
 
+    @torch.jit.ignore
+    def infer_total_number_of_predictions(self, predictions: Any) -> int:
```

Comment:
@torch.jit.ignore tells torch to ignore and not attempt to trace this function at all (torch.jit.trace & torch.onnx.export are respecting this decorator).
The idea is that we do not need this function even visible in traced module because it is not-needed there to start with. So short answer it will be dropped from ONNX completely.